toolchain/src/integration-test/resources/result.owl

```diff
@@ -0,0 +1,2248 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this file be committed at all?

Reply:
Uh, no (again)

Reply:
It is still there. Now `results.owl` is part of `.gitignore`. Should this be `result.owl` in `.gitignore`?

Reply:
I added it to the .gitignore because it will be produced by the integration test. The result.owl should not be in the repository (apparently it is called result.owl and not result**s**.owl ^^). I fixed this in the latest commit. Also, I squashed the commits so that the file should no longer appear in the history.